src/releng_docs/deploy/regular.rst

```diff
@@ -1,31 +1,73 @@
-.. _deploy-weekly-releases:
+.. _deploy-regular:
 
-Regular weekly releases
-=======================
+Regular deployment
+==================
 
-Push to production happen in weekly batches. Once a week (usually on Thursday)
-a release happens.
+Regular production deployment of all projects happens once every two weeks.
 
-.. _deploy-release-managers:
+.. _deploy-managers:
 
-Current administrators that perform this weekly release are:
+Administrators that perform this regular release are:
 
 - `Rok Garbas`_
 - `Bastien Abadie`_
 - `Rail Aliiev`_
+- `Jan Keromnes`_
 
-Protocal that we follow is:
+Release schedule is published in `Release Services calendar`_. Protocol we follow is:
 
+TODO: this should happen on Wed morning, point to the calendar
 
-#. Prior to release a push to ``staging`` branch must happen. This will
-   trigger a deploy of all projects to staging environments.
+#. Push to staging channel.
+
+   A day before pushing to production, on Wednesday morning, we push to
+   projects to **staging channel**.
+
+   To trigger automatic deployment to staging channel you need to force push
+   from ``master`` to ``staging`` branch.
 
    .. code-block:: console
 
         $ git clone git@github.com:mozilla/release-services.git
         $ cd release-services
         $ git push -f origin origin/master:staging
```

Comment:
Nit: Maybe we could remove the `$` prefix to make it easier to copy/paste these blocks?